client/src/components/AddNewItem/AddNewItem.tsx

```diff
@@ -11,18 +28,90 @@ interface Props {
 export default function AddNewItem({ lists }: Props): JSX.Element {
   const classes = useStyles();
   const [list, setList] = useState('');
+  const [openNewItemModal, setOpenNewItemModal] = useState(false);
+  const [newItemUrl, setNewItemUrl] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [snackbar, setSnackbar] = useState({
+    openSnackbar: false,
+    notification: '',
+    severity: 'info',
+  });
+  const [product, setProduct] = useState({
+    name: '',
+    price: '',
+    imageUrl: '',
+  });
+
+  function Alert(props: AlertProps) {
```

Comment:
you might want to move this to its own file if you anticipate to use it more broadly across the project :)